plugin/errors.go

```diff
@@ -65,8 +65,8 @@ func NewValidationError(msg string) *Error {
 	return NewError(ErrorCodeValidation, msg)
 }
 
-func NewValidationErrorf(format string, msg string) *Error {
-	return NewError(ErrorCodeValidation, fmt.Sprintf(format, msg))
+func NewValidationErrorf(format string, msg ...any) *Error {
```

Comment:
Since `Error` struct is also used to represent a request validation error, its description needs to be updated: line 37 & 38.